src/libraries/System.ComponentModel.TypeConverter/tests/ComponentResourceManagerTests.cs

```diff
@@ -164,10 +164,12 @@ public void ApplyResources_IComponentWithDesignModeSite_Success()
 
         private class TestSite : ISite
         {
+            private TestContainer _testContainer = new TestContainer();
+
             public bool DesignMode { get; set; }
 
             public IComponent Component => throw new NotImplementedException();
-            public IContainer Container => throw new NotImplementedException();
+            public IContainer Container => _testContainer;
```

Comment:
Maybe we should just return null here.

```suggestion
            public IContainer Container => null;
```